doc/api/tls.markdown

```diff
@@ -595,16 +600,16 @@ Creates a new client connection to the given `port` and `host` (old API) or
     CA certs of the client in PFX or PKCS12 format.
 
   - `key`: A string or `Buffer` containing the private key of the client in
-    PEM format. (Could be an array of keys).
+    PEM format. (Could be an array of keys.)
 
-  - `passphrase`: A string of passphrase for the private key or pfx.
+  - `passphrase`: A string containing the passphrase for the private key or pfx.
 
   - `cert`: A string or `Buffer` containing the certificate key of the client in
-    PEM format. (Could be an array of certs).
+    PEM format. (Could be an array of certs.)
 
   - `ca`: A string, `Buffer` or array of strings or `Buffer`s of trusted
     certificates in PEM format. If this is omitted several well known "root"
-    CAs will be used, like VeriSign. These are used to authorize connections.
+    CAs will, like VeriSign, be used. These are used to authorize connections.
```

Comment:
wouldn't it be better to put the `like Verisign` before the will here? `CAs (like Verisign) will be used.` or something like that?